messenger-appUITests/messenger_appUITests.swift

```diff
@@ -1,36 +1,207 @@
 //
-//  messenger_appUITests.swift
+//  messenger_appUITests1.swift
```

Comment:
That 1 at the end is a leftover?

Reply:
Good spot, missed updating the comment 👍